pkgs/os-specific/linux/ledger-udev-rules/default.nix

```
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation {
   pname = "ledger-udev-rules";
-  version = "unstable-2021-09-10";
+  version = "0-unstable-2024-05-15";
```

Why the `0-`?

---

Also, this should be the date of the upstream commit, so 
```suggestion
  version = "unstable-2024-02-12";
```

---

> Why the `0-`?

Thats a relatively new convention that Scrumplex mentioned in his review: https://github.com/NixOS/nixpkgs/pull/311899#discussion_r1601450578

---

> Why the `0-`?

https://github.com/NixOS/nixpkgs/pull/234201

---

> Also, this should be the date of the upstream commit, so

Ohh makes sense, i will change that but leave the 0 like Scrumplex suggested.